packages/create-plugin/src/commands/generate/prettify-files.ts

```diff
@@ -4,13 +4,13 @@ import fs from 'node:fs';
 
 const exec = promisify(nodeExec);
 
-export async function prettifyFiles(exportPath: string) {
+export async function prettifyFiles(exportPath: string, packageManagerExecute = 'npx') {
   if (!fs.existsSync(exportPath)) {
     return '';
   }
 
   try {
-    let command = 'npx -y prettier@2 . --write';
+    let command = `${packageManagerExecute} -y prettier@2 . --write`;
```

Comment:
the `-y` option will cause problem with other package managers than are not `npx`. They most likely fail because the option doesn't exist.